src/default_panels/StyleLayoutPanel.js

```diff
@@ -40,11 +40,13 @@ const StyleLayoutPanel = (props, {localize: _}) => (
     </PlotlyFold>
 
     <PlotlyFold name={_('Title')}>
-      <TextEditor attr="title.text" />
-      <FontSelector label={_('Typeface')} attr="title.font.family" clearable={false} />
-      <Numeric label={_('Font Size')} attr="title.font.size" units="px" />
-      <ColorPicker label={_('Font Color')} attr="title.font.color" />
-      <Numeric label={_('Horizontal Position')} showSlider step={0.02} attr="title.x" />
+      <PlotlySection name={_('Title')} attr="title.text">
```

Comment:
What's the point of having a section here?
![image](https://user-images.githubusercontent.com/2717746/50028584-34d45580-ffbe-11e8-9c07-692eee5bfadd.png)